### ISSUE TITLE
Remove @attr decorator from tests

### ISSUE THREAD

**Issue by elyezer** [SatelliteQE/robottelo#1820]
`@attr` is a decorator which nose provides, as robottelo does not stick with a specific test runner then this decorator should be removed. There is no practical advantage of using this decorator because it adds overhead to our development and are not being used.

If there is any reason to leave it, then leave a comment. But remember that we will still be able to select the tests because they just filter tests from one module to another. With that said, we can just run `nose tests/foreman/cli/test_activationkey.py` and it will run cli activation key tests.

Current entries are:

```
tests/foreman/cli/test_activationkey.py:    @attr('cli', 'activation-key')
tests/foreman/cli/test_activationkey.py:    @attr('cli', 'activation-key')
tests/foreman/cli/test_activationkey.py:    @attr('cli', 'activation-key')
tests/foreman/cli/test_activationkey.py:    @attr('cli', 'activation-key')
tests/foreman/cli/test_activationkey.py:    @attr('cli', 'activation-key')
tests/foreman/cli/test_contenthost.py:    @attr('cli', 'content-host')
tests/foreman/cli/test_contenthost.py:    @attr('cli', 'content-host')
tests/foreman/cli/test_contenthost.py:    @attr('cli', 'content-host')
tests/foreman/cli/test_contenthost.py:    @attr('cli', 'content-host')
tests/foreman/cli/test_contenthost.py:    @attr('cli', 'content-host')
tests/foreman/cli/test_contenthost.py:    @attr('cli', 'content-host')
tests/foreman/cli/test_contenthost.py:    @attr('cli', 'content-host')
tests/foreman/cli/test_contenthost.py:    @attr('cli', 'content-host')
tests/foreman/cli/test_contenthost.py:    @attr('cli', 'content-host')
tests/foreman/cli/test_contenthost.py:    @attr('cli', 'content-host')
tests/foreman/cli/test_contenthost.py:    @attr('cli', 'content-host')
tests/foreman/cli/test_contenthost.py:    @attr('cli', 'content-host')
tests/foreman/cli/test_contenthost.py:    @attr('cli', 'content-host')
tests/foreman/cli/test_fact.py:    @attr('cli', 'fact')
tests/foreman/cli/test_fact.py:    @attr('cli', 'fact')
tests/foreman/cli/test_host_collection.py:    @attr('cli', 'hostcollection')
tests/foreman/cli/test_host_collection.py:    @attr('cli', 'hostcollection')
tests/foreman/cli/test_host_collection.py:    @attr('cli', 'hostcollection')
tests/foreman/cli/test_host_collection.py:    @attr('cli', 'hostcollection')
tests/foreman/cli/test_host_collection.py:    @attr('cli', 'hostcollection')
tests/foreman/cli/test_host_collection.py:    @attr('cli', 'hostcollection')
tests/foreman/cli/test_host_collection.py:    @attr('cli', 'hostcollection')
tests/foreman/cli/test_host_collection.py:    @attr('cli', 'hostcollection')
tests/foreman/cli/test_product.py:    @attr('cli', 'product')
tests/foreman/cli/test_product.py:    @attr('cli', 'product')
tests/foreman/cli/test_product.py:    @attr('cli', 'product')
tests/foreman/cli/test_product.py:    @attr('cli', 'product')
tests/foreman/cli/test_product.py:    @attr('cli', 'product')
tests/foreman/cli/test_product.py:    @attr('cli', 'product')
tests/foreman/cli/test_product.py:    @attr('cli', 'product')
tests/foreman/cli/test_product.py:    @attr('cli', 'product')
tests/foreman/cli/test_product.py:    @attr('cli', 'product')
tests/foreman/cli/test_product.py:    @attr('cli', 'product')
tests/foreman/cli/test_product.py:    @attr('cli', 'product')
tests/foreman/cli/test_product.py:    @attr('cli', 'product')
tests/foreman/cli/test_product.py:    @attr('cli', 'product')
tests/foreman/cli/test_repository.py:    @attr('cli', 'repository')
tests/foreman/cli/test_repository.py:    @attr('cli', 'repository')
tests/foreman/cli/test_repository.py:    @attr('cli', 'repository')
tests/foreman/cli/test_repository.py:    @attr('cli', 'repository')
tests/foreman/cli/test_repository.py:    @attr('cli', 'repository')
tests/foreman/cli/test_repository.py:    @attr('cli', 'repository')
tests/foreman/cli/test_repository.py:    @attr('cli', 'repository')
tests/foreman/cli/test_repository.py:    @attr('cli', 'repository')
tests/foreman/cli/test_repository.py:    @attr('cli', 'repository', 'docker')
tests/foreman/cli/test_repository.py:    @attr('cli', 'repository')
tests/foreman/cli/test_repository.py:    @attr('cli', 'repository')
tests/foreman/cli/test_repository.py:    @attr('cli', 'repository', 'docker')
tests/foreman/cli/test_repository.py:    @attr('cli', 'repository')
tests/foreman/cli/test_repository.py:    @attr('cli', 'repository')
tests/foreman/cli/test_repository.py:    @attr('cli', 'repository')
tests/foreman/cli/test_repository.py:    @attr('cli', 'repository')
tests/foreman/cli/test_subnet.py:    @attr('cli', 'subnet')
tests/foreman/cli/test_subnet.py:    @attr('cli', 'subnet')
tests/foreman/cli/test_subnet.py:    @attr('cli', 'subnet')
tests/foreman/cli/test_subnet.py:    @attr('cli', 'subnet')
tests/foreman/cli/test_syncplan.py:    @attr('cli', 'syncplan')
tests/foreman/cli/test_syncplan.py:    @attr('cli', 'syncplan')
tests/foreman/cli/test_syncplan.py:    @attr('cli', 'syncplan')
tests/foreman/cli/test_syncplan.py:    @attr('cli', 'syncplan')
tests/foreman/cli/test_syncplan.py:    @attr('cli', 'syncplan')
tests/foreman/cli/test_syncplan.py:    @attr('cli', 'syncplan')
tests/foreman/cli/test_syncplan.py:    @attr('cli', 'syncplan')
tests/foreman/cli/test_syncplan.py:    @attr('cli', 'syncplan')
tests/foreman/smoke/test_api_smoke.py:    @attr('smoke')
tests/foreman/smoke/test_api_smoke.py:    @attr('smoke')
tests/foreman/smoke/test_api_smoke.py:    @attr('smoke')
tests/foreman/smoke/test_api_smoke.py:    @attr('smoke')
tests/foreman/smoke/test_api_smoke.py:    @attr('smoke')
tests/foreman/smoke/test_api_smoke.py:    @attr('smoke')
tests/foreman/smoke/test_cli_smoke.py:    @attr('smoke')
tests/foreman/smoke/test_cli_smoke.py:    @attr('smoke')
tests/foreman/smoke/test_cli_smoke.py:    @attr('smoke')
tests/foreman/smoke/test_cli_smoke.py:    @attr('smoke')
tests/foreman/smoke/test_ui_smoke.py:    @attr('smoke')
tests/foreman/smoke/test_ui_smoke.py:    @attr('smoke')
tests/foreman/smoke/test_ui_smoke.py:    @attr('smoke')
tests/foreman/smoke/test_ui_smoke.py:    @attr('smoke')
tests/foreman/ui/test_activationkey.py:    @attr('ui', 'ak', 'implemented')
tests/foreman/ui/test_activationkey.py:    @attr('ui', 'ak', 'implemented')
tests/foreman/ui/test_activationkey.py:    @attr('ui', 'ak', 'implemented')
tests/foreman/ui/test_activationkey.py:    @attr('ui', 'ak', 'implemented')
tests/foreman/ui/test_activationkey.py:    @attr('ui', 'ak', 'implemented')
tests/foreman/ui/test_activationkey.py:    @attr('ui', 'ak', 'implemented')
tests/foreman/ui/test_activationkey.py:    @attr('ui', 'ak', 'implemented')
tests/foreman/ui/test_activationkey.py:    @attr('ui', 'ak', 'implemented')
tests/foreman/ui/test_activationkey.py:    @attr('ui', 'ak', 'implemented')
tests/foreman/ui/test_activationkey.py:    @attr('ui', 'ak', 'implemented')
tests/foreman/ui/test_activationkey.py:    @attr('ui', 'ak', 'implemented')
tests/foreman/ui/test_activationkey.py:    @attr('ui', 'ak', 'implemented')
tests/foreman/ui/test_activationkey.py:    @attr('ui', 'ak', 'implemented')
tests/foreman/ui/test_activationkey.py:    @attr('ui', 'ak', 'implemented')
tests/foreman/ui/test_activationkey.py:    @attr('ui', 'ak', 'implemented')
tests/foreman/ui/test_activationkey.py:    @attr('ui', 'ak', 'implemented')
tests/foreman/ui/test_activationkey.py:    @attr('ui', 'ak', 'implemented')
tests/foreman/ui/test_activationkey.py:    @attr('ui', 'ak', 'implemented')
tests/foreman/ui/test_activationkey.py:    @attr('ui', 'ak', 'implemented')
tests/foreman/ui/test_computeresource.py:    @attr('ui', 'resource', 'implemented')
tests/foreman/ui/test_computeresource.py:    @attr('ui', 'resource', 'implemented')
tests/foreman/ui/test_computeresource.py:    @attr('ui', 'resource', 'implemented')
tests/foreman/ui/test_computeresource.py:    @attr('ui', 'resource', 'implemented')
tests/foreman/ui/test_computeresource.py:    @attr('ui', 'resource', 'implemented')
tests/foreman/ui/test_computeresource.py:    @attr('ui', 'resource', 'implemented')
tests/foreman/ui/test_computeresource.py:    @attr('ui', 'location', 'implemented')
tests/foreman/ui/test_config_groups.py:    @attr('ui', 'config-groups', 'implemented')
tests/foreman/ui/test_config_groups.py:    @attr('ui', 'config-groups', 'implemented')
tests/foreman/ui/test_config_groups.py:    @attr('ui', 'config-groups', 'implemented')
tests/foreman/ui/test_config_groups.py:    @attr('ui', 'config-group', 'implemented')
tests/foreman/ui/test_config_groups.py:    @attr('ui', 'config-groups', 'implemented')
tests/foreman/ui/test_contentenv.py:    @attr('ui', 'contentenv', 'implemented')
tests/foreman/ui/test_contentenv.py:    @attr('ui', 'contentenv', 'implemented')
tests/foreman/ui/test_contentenv.py:    @attr('ui', 'contentenv', 'implemented')
tests/foreman/ui/test_contentenv.py:    @attr('ui', 'contentenv', 'implemented')
tests/foreman/ui/test_domain.py:    @attr('ui', 'domain', 'implemented')
tests/foreman/ui/test_domain.py:    @attr('ui', 'domain', 'implemented')
tests/foreman/ui/test_domain.py:    @attr('ui', 'domain', 'implemented')
tests/foreman/ui/test_domain.py:    @attr('ui', 'domain', 'implemented')
tests/foreman/ui/test_domain.py:    @attr('ui', 'domain', 'implemented')
tests/foreman/ui/test_domain.py:    @attr('ui', 'domain', 'implemented')
tests/foreman/ui/test_domain.py:    @attr('ui', 'domain', 'implemented')
tests/foreman/ui/test_environment.py:    @attr('ui', 'environment', 'implemented')
tests/foreman/ui/test_environment.py:    @attr('ui', 'environment', 'implemented')
tests/foreman/ui/test_environment.py:    @attr('ui', 'environment', 'implemented')
tests/foreman/ui/test_environment.py:    @attr('ui', 'environment', 'implemented')
tests/foreman/ui/test_environment.py:    @attr('ui', 'environment', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_gpgkey.py:    @attr('ui', 'gpgkey', 'implemented')
tests/foreman/ui/test_hw_model.py:    @attr('ui', 'hardware-model', 'implemented')
tests/foreman/ui/test_hw_model.py:    @attr('ui', 'hardware-model', 'implemented')
tests/foreman/ui/test_hw_model.py:    @attr('ui', 'hardware-model', 'implemented')
tests/foreman/ui/test_hw_model.py:    @attr('ui', 'hardware-model', 'implemented')
tests/foreman/ui/test_hw_model.py:    @attr('ui', 'hardware-model', 'implemented')
tests/foreman/ui/test_location.py:    @attr('ui', 'location', 'implemented')
tests/foreman/ui/test_location.py:    @attr('ui', 'location', 'implemented')
tests/foreman/ui/test_location.py:    @attr('ui', 'location', 'implemented')
tests/foreman/ui/test_location.py:    @attr('ui', 'location', 'implemented')
tests/foreman/ui/test_location.py:    @attr('ui', 'location', 'implemented')
tests/foreman/ui/test_location.py:    @attr('ui', 'location', 'implemented')
tests/foreman/ui/test_location.py:    @attr('ui', 'location', 'implemented')
tests/foreman/ui/test_location.py:    @attr('ui', 'location', 'implemented')
tests/foreman/ui/test_location.py:    @attr('ui', 'location', 'implemented')
tests/foreman/ui/test_location.py:    @attr('ui', 'location', 'implemented')
tests/foreman/ui/test_location.py:    @attr('ui', 'location', 'implemented')
tests/foreman/ui/test_location.py:    @attr('ui', 'location', 'implemented')
tests/foreman/ui/test_location.py:    @attr('ui', 'location', 'implemented')
tests/foreman/ui/test_location.py:    @attr('ui', 'location', 'implemented')
tests/foreman/ui/test_location.py:    @attr('ui', 'location', 'implemented')
tests/foreman/ui/test_location.py:    @attr('ui', 'location', 'implemented')
tests/foreman/ui/test_location.py:    @attr('ui', 'location', 'implemented')
tests/foreman/ui/test_location.py:    @attr('ui', 'location', 'implemented')
tests/foreman/ui/test_location.py:    @attr('ui', 'location', 'implemented')
tests/foreman/ui/test_location.py:    @attr('ui', 'location', 'implemented')
tests/foreman/ui/test_location.py:    @attr('ui', 'location', 'implemented')
tests/foreman/ui/test_location.py:    @attr('ui', 'location', 'implemented')
tests/foreman/ui/test_location.py:    @attr('ui', 'location', 'implemented')
tests/foreman/ui/test_location.py:    @attr('ui', 'location', 'implemented')
tests/foreman/ui/test_org.py:    @attr('ui', 'org', 'implemented')
tests/foreman/ui/test_org.py:    @attr('ui', 'org', 'implemented')
tests/foreman/ui/test_org.py:    @attr('ui', 'org', 'implemented')
tests/foreman/ui/test_org.py:    @attr('ui', 'org', 'implemented')
tests/foreman/ui/test_org.py:    @attr('ui', 'org', 'implemented')
tests/foreman/ui/test_org.py:    @attr('ui', 'org', 'implemented')
tests/foreman/ui/test_org.py:    @attr('ui', 'org', 'implemented')
tests/foreman/ui/test_org.py:    @attr('ui', 'org', 'implemented')
tests/foreman/ui/test_org.py:    @attr('ui', 'org', 'implemented')
tests/foreman/ui/test_org.py:    @attr('ui', 'org', 'implemented')
tests/foreman/ui/test_org.py:    @attr('ui', 'org', 'implemented')
tests/foreman/ui/test_org.py:    @attr('ui', 'org', 'implemented')
tests/foreman/ui/test_org.py:    @attr('ui', 'org', 'implemented')
tests/foreman/ui/test_org.py:    @attr('ui', 'org', 'implemented')
tests/foreman/ui/test_org.py:    @attr('ui', 'org', 'implemented')
tests/foreman/ui/test_org.py:    @attr('ui', 'org', 'implemented')
tests/foreman/ui/test_org.py:    @attr('ui', 'org', 'implemented')
tests/foreman/ui/test_org.py:    @attr('ui', 'org', 'implemented')
tests/foreman/ui/test_org.py:    @attr('ui', 'org', 'implemented')
tests/foreman/ui/test_org.py:    @attr('ui', 'org', 'implemented')
tests/foreman/ui/test_org.py:    @attr('ui', 'org', 'implemented')
tests/foreman/ui/test_org.py:    @attr('ui', 'org', 'implemented')
tests/foreman/ui/test_org.py:    @attr('ui', 'org', 'implemented')
tests/foreman/ui/test_org.py:    @attr('ui', 'org', 'implemented')
tests/foreman/ui/test_org.py:    @attr('ui', 'org', 'implemented')
tests/foreman/ui/test_org.py:    @attr('ui', 'org', 'implemented')
tests/foreman/ui/test_org.py:    @attr('ui', 'org', 'implemented')
tests/foreman/ui/test_org.py:    @attr('ui', 'org', 'implemented')
tests/foreman/ui/test_org.py:    @attr('ui', 'org', 'implemented')
tests/foreman/ui/test_products.py:    @attr('ui', 'prd', 'implemented')
tests/foreman/ui/test_products.py:    @attr('ui', 'prd', 'implemented')
tests/foreman/ui/test_products.py:    @attr('ui', 'prd', 'implemented')
tests/foreman/ui/test_products.py:    @attr('ui', 'prd', 'implemented')
tests/foreman/ui/test_products.py:    @attr('ui', 'prd', 'implemented')
tests/foreman/ui/test_products.py:    @attr('ui', 'prd', 'implemented')
tests/foreman/ui/test_products.py:    @attr('ui', 'prd', 'implemented')
tests/foreman/ui/test_puppet_classes.py:    @attr('ui', 'puppet-classes', 'implemented')
tests/foreman/ui/test_puppet_classes.py:    @attr('ui', 'puppet-classes', 'implemented')
tests/foreman/ui/test_puppet_classes.py:    @attr('ui', 'puppet-classes', 'implemented')
tests/foreman/ui/test_puppet_classes.py:    @attr('ui', 'puppet-classes', 'implemented')
tests/foreman/ui/test_puppet_classes.py:    @attr('ui', 'puppet-classes', 'implemented')
tests/foreman/ui/test_repository.py:    @attr('ui', 'repo', 'implemented')
tests/foreman/ui/test_repository.py:    @attr('ui', 'repo', 'implemented')
tests/foreman/ui/test_repository.py:    @attr('ui', 'repo', 'implemented', 'docker')
tests/foreman/ui/test_repository.py:    @attr('ui', 'repo', 'implemented', 'docker')
tests/foreman/ui/test_repository.py:    @attr('ui', 'repo', 'implemented')
tests/foreman/ui/test_repository.py:    @attr('ui', 'repo', 'implemented')
tests/foreman/ui/test_repository.py:    @attr('ui', 'repo', 'implemented')
tests/foreman/ui/test_repository.py:    @attr('ui', 'repo', 'implemented')
tests/foreman/ui/test_repository.py:    @attr('ui', 'repo', 'implemented')
tests/foreman/ui/test_repository.py:    @attr('ui', 'repo', 'implemented')
tests/foreman/ui/test_repository.py:    @attr('ui', 'repo', 'implemented')
tests/foreman/ui/test_repository.py:    @attr('ui', 'repo', 'implemented')
tests/foreman/ui/test_repository.py:    @attr('ui', 'repo', 'implemented')
tests/foreman/ui/test_repository.py:    @attr('ui', 'repo', 'implemented')
tests/foreman/ui/test_subnet.py:    @attr('ui', 'subnet', 'implemented')
tests/foreman/ui/test_subnet.py:    @attr('ui', 'subnet', 'implemented')
tests/foreman/ui/test_subnet.py:    @attr('ui', 'subnet', 'implemented')
tests/foreman/ui/test_subnet.py:    @attr('ui', 'subnet', 'implemented')
tests/foreman/ui/test_subnet.py:    @attr('ui', 'subnet', 'implemented')
tests/foreman/ui/test_subnet.py:    @attr('ui', 'subnet', 'implemented')
tests/foreman/ui/test_subnet.py:    @attr('ui', 'subnet', 'implemented')
tests/foreman/ui/test_subnet.py:    @attr('ui', 'subnet', 'implemented')
tests/foreman/ui/test_subnet.py:    @attr('ui', 'subnet', 'implemented')
tests/foreman/ui/test_subscription.py:    @attr('ui', 'subs', 'implemented')
tests/foreman/ui/test_subscription.py:    @attr('ui', 'subs', 'implemented')
tests/foreman/ui/test_subscription.py:    @attr('ui', 'subs', 'implemented')
tests/foreman/ui/test_sync.py:    @attr('ui', 'sync', 'implemented')
tests/foreman/ui/test_syncplan.py:    @attr('ui', 'syncplan', 'implemented')
tests/foreman/ui/test_syncplan.py:    @attr('ui', 'syncplan', 'implemented')
tests/foreman/ui/test_syncplan.py:    @attr('ui', 'syncplan', 'implemented')
tests/foreman/ui/test_syncplan.py:    @attr('ui', 'syncplan', 'implemented')
tests/foreman/ui/test_syncplan.py:    @attr('ui', 'syncplan', 'implemented')
tests/foreman/ui/test_syncplan.py:    @attr('ui', 'syncplan', 'implemented')
tests/foreman/ui/test_syncplan.py:    @attr('ui', 'syncplan', 'implemented')
tests/foreman/ui/test_syncplan.py:    @attr('ui', 'syncplan', 'implemented')
tests/foreman/ui/test_syncplan.py:    @attr('ui', 'syncplan', 'implemented')
tests/foreman/ui/test_syncplan.py:    @attr('ui', 'syncplan', 'implemented')
tests/foreman/ui/test_user.py:    @attr('ui', 'user', 'implemented')
tests/foreman/ui/test_user.py:    @attr('ui', 'user', 'implemented')
tests/foreman/ui/test_user.py:    @attr('ui', 'user', 'implemented')
tests/foreman/ui/test_user.py:    @attr('ui', 'user', 'implemented')
tests/foreman/ui/test_user.py:    @attr('ui', 'user', 'implemented')
tests/foreman/ui/test_user.py:    @attr('ui', 'user', 'implemented')
tests/foreman/ui/test_user.py:    @attr('ui', 'user', 'implemented')
tests/foreman/ui/test_user.py:    @attr('ui', 'user', 'implemented')
tests/foreman/ui/test_user.py:    @attr('ui', 'user', 'implemented')
tests/foreman/ui/test_user.py:    @attr('ui', 'user', 'implemented')
tests/foreman/ui/test_user.py:    @attr('ui', 'user', 'implemented')
tests/foreman/ui/test_user.py:    @attr('ui', 'user', 'implemented')
tests/foreman/ui/test_user.py:    @attr('ui', 'user', 'implemented')
tests/foreman/ui/test_user.py:    @attr('ui', 'user', 'implemented')
tests/foreman/ui/test_user.py:    @attr('ui', 'user', 'implemented')
tests/foreman/ui/test_user.py:    @attr('ui', 'user', 'implemented')
tests/foreman/ui/test_user.py:    @attr('ui', 'user', 'implemented')
tests/foreman/ui/test_user.py:    @attr('ui', 'user', 'implemented')
tests/foreman/ui/test_user.py:    @attr('ui', 'user', 'implemented')
tests/foreman/ui/test_user.py:    @attr('ui', 'user', 'implemented')
```

**Comment by Ichimonji10** [comment]
:+1:

The decorator is not being used, and it's easy to select tests based on their location in the test suite. I don't see the need to use `@attr` right now.

**Comment by elyezer** [comment]
Fixed on #1851, closing this.
